internal/dag/httpproxy_processor.go

```diff
@@ -1390,9 +1399,10 @@ func toStringSlice(hvs []contour_api_v1.CORSHeaderValue) []string {
 	return s
 }
 
-func includeMatchConditionsIdentical(include contour_api_v1.Include, seenConds map[string][][]HeaderMatchCondition) bool {
+func includeMatchConditionsIdentical(include contour_api_v1.Include, seenHeaderConds map[string][][]HeaderMatchCondition, seenQueryParamConds map[string][][]QueryParamMatchCondition) bool {
```

Comment:
it would be great to have some coverage over the logic here similar to this test case and a few below it: https://github.com/projectcontour/contour/blob/052a12d88decdb068926333b9f90ae77cb650a5c/internal/dag/status_test.go#L1731

Reply:
Good catch, totally missed this one!